Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM centos:7
```

Comment:
Because alpine has some weird versions of grep and other tools that do not support all flags and ubuntu/debian is too overbloated.